robber/expect.py

```diff
@@ -19,8 +19,9 @@ class expect:
     matchers = {}
     message = None
 
-    def __init__(self, object):
-        self.object = object
+    def __init__(self, obj):
+        self.obj = obj
+        self.is_negative = False
```

Comment:
I don't say about the trend here, since `negate` is not very popular, of course. But the meaning of it is something like the reversing the matcher's meaning, which is obviously closer to what we are going to do. Because logically, you can do something like not_to.not_to, it contains `not` but actually it's not negative, it's just negated twice. Anyway, it's not too bad then.